primer/brand
# Add fullWidth prop to Card component

## Summary

Closes #603 

Adds a `fullWidth` prop to the Card component to allow the card to take up the full width of the parent container. 

To date, we've expected end-users to override the value of `--brand-Card-maxWidth` to be 100%. We're seeing this happen too frequently, and this should now be supported in the API itself as a full width card is a different use case than a card with a custom width. 

## List of notable changes:



**added** `fullWidth` for Card component, along with tests and documentation

## What should reviewers focus on?



- Any visual design reasons for not enabling this in the API

## Steps to test:



1. Open the following story 
2. Increase/decrease window size
3. Verify that the Card stretches to the full width of the parent container, unlike the default card

## Supporting resources (related issues, external links, etc):

- Original request #603 

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2024-06-12 at 15 23 59](https://github.com/primer/brand/assets/13340707/ad7000ff-9f22-44c8-bda2-c0430c3864ec)


 </td>
<td valign="top">

![Screenshot 2024-06-12 at 15 23 44](https://github.com/primer/brand/assets/13340707/6f0cb58a-6823-4d00-85b0-317d6e4dfeac)


</td>
</tr>
</table>
